packages/adapter-d1/README.md

```diff
@@ -1,3 +1,79 @@
 # Prisma driver adapter for Cloudflare D1
 
-**INTERNAL PACKAGE, DO NOT USE**
+Prisma driver adapter for [Cloudflare's Serverless Driver D1](https://developers.cloudflare.com/d1/). Refer to the [announcement blog post](https://prisma.io/cloudflare-d1) and our [docs](https://www.prisma.io/docs/guides/database/cloudflare-d1) for more details.
```

Comment:
Blog post and docs will only happen later.
```suggestion
Prisma driver adapter for [Cloudflare's Serverless Driver D1](https://developers.cloudflare.com/d1/).
```

Reply:
So just remove in the meantime?

Reply:
Yes. Or comment out.